src/main/java/org/sagebionetworks/bridge/models/DateRangeResourceList.java

```diff
@@ -9,22 +9,29 @@
 
 public class DateRangeResourceList<T> extends ResourceList<T> {
     
+    public DateRangeResourceList(List<T> items, boolean suppressDeprecated) {
+        super(items, suppressDeprecated);
+    }
```

Comment:
Did we forget to include a unit test for this class?

Reply:
Yes, added it.